.circleci/config.yml

```diff
@@ -107,7 +107,9 @@ commands:
           command: |
             if [ ${CIRCLE_BRANCH} != "" ] &&
               [ ${CIRCLE_BRANCH} != "master" ] &&
-              ! git diff --name-only $(git merge-base origin/master ${CIRCLE_BRANCH}) | grep -qvE '(.md$)|(.pdf$)'; then
+              ! git diff --name-only $(git merge-base origin/master ${CIRCLE_BRANCH}) \
+                | grep -qvE '(.md)|(.pdf)|(.png)|(.svg)|(Test.java)|(Test.kt)|(.mp4)|(.log)|(.txt)'; then
```

Comment:
Can you put all these extensions into an environment var called `E2E_IGNORE_PATTERNS`? Can we use this approach for all our jobs?